src/script/media/MediaDevicesHandler.ts

```diff
@@ -249,9 +263,11 @@ export class MediaDevicesHandler {
 
   /**
    * Update list of available MediaDevices.
+   * @param [camera=false] If `camera=true`, a video track is also created when the device list is read.
+   * This ensures that the video device labels can also be read. This is necessary for initializing the entire device list.
    * @returns Resolves with all MediaDevices when the list has been updated
    */
-  public async refreshMediaDevices(): Promise<MediaDeviceInfo[]> {
+  public async refreshMediaDevices(camera: boolean = false): Promise<MediaDeviceInfo[]> {
```

Comment:
You don't have to assign type there, because default value assign type to arg ;)

```suggestion
  public async refreshMediaDevices(camera = false): Promise<MediaDeviceInfo[]> {
```